tests/toranj/build.sh

```diff
@@ -206,129 +187,126 @@ case ${build_config} in
         echo "==================================================================================================="
         cd "${top_builddir}" || die "cd failed"
-        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${ot_coverage} \
+        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${coverage} \
+            -DOT_DOC=OFF "-DBUILD_TESTING=$tests" -DOT_LOG_LEVEL_DYNAMIC=ON \
+            -DOT_MAC_FILTER=ON \
             -DOT_APP_CLI=ON -DOT_APP_NCP=OFF -DOT_APP_RCP=OFF \
             -DOT_15_4=ON -DOT_TREL=OFF \
             -DOT_CONFIG=../tests/toranj/openthread-core-toranj-config-simulation.h \
             "${top_srcdir}" || die
         ninja || die
-        cp -p ${top_builddir}/examples/apps/cli/ot-cli-ftd ${top_builddir}/examples/apps/cli/ot-cli-ftd-15.4
+        cp -vp ${top_builddir}/examples/apps/cli/ot-cli-ftd ${top_builddir}/examples/apps/cli/ot-cli-ftd-15.4
         ;;
 
     cli-trel)
         echo "==================================================================================================="
         echo "Building OpenThread CLI FTD mode with simulation platform - TREL radio"
         echo "==================================================================================================="
         cd "${top_builddir}" || die "cd failed"
-        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${ot_coverage} \
+        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${coverage} \
+            -DOT_DOC=OFF "-DBUILD_TESTING=$tests" -DOT_LOG_LEVEL_DYNAMIC=ON \
+            -DOT_MAC_FILTER=ON \
             -DOT_APP_CLI=ON -DOT_APP_NCP=OFF -DOT_APP_RCP=OFF \
             -DOT_15_4=OFF -DOT_TREL=ON \
             -DOT_CONFIG=../tests/toranj/openthread-core-toranj-config-simulation.h \
             "${top_srcdir}" || die
         ninja || die
-        cp -p ${top_builddir}/examples/apps/cli/ot-cli-ftd ${top_builddir}/examples/apps/cli/ot-cli-ftd-trel
+        cp -vp ${top_builddir}/examples/apps/cli/ot-cli-ftd ${top_builddir}/examples/apps/cli/ot-cli-ftd-trel
         ;;
 
     cli-15.4+trel | cli-trel+15.4)
         echo "==================================================================================================="
         echo "Building OpenThread NCP FTD mode with simulation platform - multi radio (15.4 + TREL)"
         echo "==================================================================================================="
         cd "${top_builddir}" || die "cd failed"
-        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${ot_coverage} \
+        cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${coverage} \
+            -DOT_DOC=OFF "-DBUILD_TESTING=$tests" -DOT_LOG_LEVEL_DYNAMIC=ON \
+            -DOT_MAC_FILTER=ON \
```

Comment:
Why adding the flags `-DOT_LOG_LEVEL_DYNAMIC=ON -DOT_MAC_FILTER=ON` here?

We have the `openthead-core-toran-configj.h` header file which defines all configs for toranj tests. It already defines both of these.

Reply:
I see the same below in all other changes, should remove them from all?

Reply:
Is this addresses in new push? Looks like it same as before?

Reply:
No, I'm still working on this PR. I'll mark it as draft before all tests pass again.

Reply:
Sounds good. I mark these as resolved.